javac-plugin/build.gradle.kts

```diff
@@ -17,7 +17,8 @@ dependencies {
 }
 
 tasks.compileTestJava {
-    classpath += rootProject.tasks["shadowJar"].outputs.files
+    classpath += project(":java-api").tasks.getByName("testJar").outputs.files +
+            files(rootProject.sourceSets.map { it.compileClasspath }.flatten())
```

Comment:
second part could be `rootProject.tasks["jar"]outputs.files` I think

Reply:
also I would've passed the api jar as a flag, instead of directly putting it on the classpath. like I do in the other testing project

Reply:
yeah the second one is a good idea